src/equations/compressible_euler_2d.jl

```diff
@@ -1,11 +1,22 @@
+# By default, Julia/LLVM does not use FMAs. Hence, we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi/
+@muladd begin
+
 
 @doc raw"""
     CompressibleEulerEquations2D
 
-The compressible Euler equations for an ideal gas in two space dimensions.
+The compressible Euler equations for an ideal gas with ratio of specific heats `γ`
+in two space dimensions.
 """
 struct CompressibleEulerEquations2D{RealT<:Real} <: AbstractCompressibleEulerEquations{2, 4}
   gamma::RealT
+  inv_γm1::RealT
```

Comment:
I have to admit that I don't particularly like this kind of optimization, and to be honest, I am very doubtful whether we should include it. I know the benefits, I know that this will shave off a few percent of the run time, and I've seen this kind of optimization used before in other codes.

However, even though this modification seems very minor at the surface, it makes the code *qualitatively* harder to understand for someone not experienced with it, especially for those who read it for the first time (and for whom thus almost everything is new information). Before, everything was essentially physics + math just put into Julia syntax. Now, you start adding magic names that have to be looked up, understood, and memorized, which is a significant difference for someone who just casually reads the code. For example, in FLEXI/FLUXO this was (is?) used generously, and to me it made many of the algorithms much less attractive to read and copy, because everything was encoded in this obscure notation.

To summarize, at the moment I think that this is a micro optimization for which we have no real burning platform, and which on the other hand increases the complexity of the code. I would thus argue to not include it right now in Trixi, but maybe add an issue that describes this potential optimization.

Reply:
Would you change your mind if we called this field `inv_gamma_minus_one` or something like that? From my point of view, we definitely need to avoid cryptic names that appear in quite some other codes... However, Julia has some nice editors that can help a lot. For example, hitting Ctrl+Shift+F10 in VS Code with the Julia plugin, I get
![image](https://user-images.githubusercontent.com/12693098/122449675-6b7ff200-cfa6-11eb-8baf-679b850c1bf8.png)

Reply:
TBH, I am really undecided. Would you mind leaving the final decision until after the next Trixi meeting?

Reply:
Could you add it to the next meeting agenda, please?

Reply:
![image](https://user-images.githubusercontent.com/3637659/123125595-edf62f00-d448-11eb-8c06-8120203d2bf9.png)

Reply:
Decided: Leave it as it is (with `inv_gamma_minus_one`)